src/soilId/soilIdSlice.ts

```diff
@@ -78,15 +79,20 @@ export type SoilData = Omit<
   depthIntervals: SoilDataDepthInterval[];
   depthDependentData: DepthDependentSoilData[];
 };
+
+export type LabelOptional<Type extends { label: string }> = Optional<
+  Type,
+  'label'
+>;
```

Comment:
nit: the empty string is meant to model the absence of a label, so that there aren't two possible states for no label (empty or undefined)

Reply:
I see. I was using the "missing label" to keep track of whether a soil interval was a default interval generated from a project preset. I guess it makes more sense to generate an empty string in that case, and track that state in the "mutable" option, which should probably be renamed to "projectPresetInterval" to account for that.